examples/CMakeLists.txt

```diff
@@ -269,6 +269,19 @@ if(LINUX)
         COMMAND ${EXAMPLE_NAME}
         WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
 
+    if(UMF_BUILD_LIBUMF_POOL_JEMALLOC)
```

Comment:
we should use UMF_POOL_JEMALLOC_ENABLED here

Reply:
Done